ironfish/src/rpc/routes/fees/estimateFee.test.ts

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { useBlockWithTx } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { EstimateFeeRequest } from './estimateFee'
+
+describe('estimate Fee', () => {
+  const routeTest = createRouteTest(true)
+
+  beforeAll(async () => {
+    await routeTest.node.wallet.createAccount('existingAccount', true)
+    const node = routeTest.node
+    const { block } = await useBlockWithTx(node, undefined, undefined, true, {
+      fee: 1,
+    })
+    await node.chain.addBlock(block)
+    await node.wallet.updateHead()
+  })
+
```

Comment:
It doesn't look like there is a test in here for the different priority levels, do they exist already elsewhere?

Reply:
Yeah, the test with different priority level is in [https://github.com/iron-fish/ironfish/ironfish/src/memPool/feeEstimator.test.ts](url)